makefile

```
@@ -25,7 +25,17 @@ build: clear clean-build install
 dev: clean-build clear install
 	$(call banner, "gateway is starting")
 	$(call log, "starting development server")
-	@yarn dev
+	@(set -a && source .env && yarn watch:server & yarn watch & wait)
```

Ah! Another small change, we don't need the `set -a && source .env` anymore, as we have a webpack plugin which does that for us now. 

https://github.com/guardian/gateway/blob/60459676fd596288e48dad583a9487b6b4f754a1/webpack.config.js#L63

```suggestion
	@(yarn watch:server & yarn watch & wait)
```

---

ah just removed this :)